lib/packages/core/lib/src/utils/preferences/preferences.dart

```diff
@@ -1,57 +1,37 @@
 import 'package:likeminds_chat_fl/likeminds_chat_fl.dart';
 import 'package:likeminds_chat_flutter_core/likeminds_chat_flutter_core.dart';
 
-abstract class ILMPreferenceService {
-  void storeUserData(User user);
-  User? getUser();
-  void clearUserData();
-  void storeMemberRights(MemberStateResponse memberState);
-  MemberStateResponse? getMemberRights();
-  bool fetchMemberRight(int id);
-  void clearMemberRights();
-  void storeCommunityData(Community community);
-  Community? getCommunity();
-  void clearCommunityData();
-  void clearLocalPrefs();
-}
-
-class LMChatPreferences extends ILMPreferenceService {
+class LMChatLocalPreference {
   // Singleton instance of LMChatPreference class
-  static LMChatPreferences? _instance;
-  static LMChatPreferences get instance => _instance ??= LMChatPreferences._();
-
-  LMChatPreferences._();
+  static LMChatLocalPreference? _instance;
+  static LMChatLocalPreference get instance =>
+      _instance ??= LMChatLocalPreference._();
 
-  get getCurrentUser => getUser();
+  LMChatLocalPreference._();
 
-  @override
-  Future<void> storeUserData(User user) async {
-    await LMChatCore.instance.lmChatClient.deleteLoggedInUser();
+  Future<void> storeUserData1(User user) async {
```

Comment:
Let's change the name of the function back

Reply:
done